WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -558,6 +556,14 @@ import WordPressComAnalytics
             presentViewController(alertController, animated: true, completion: nil)
         }
     }
+    
+    private func sharePost(postID: NSManagedObjectID, fromView anchorView: UIView) {
+        if let post = self.postWithObjectID(postID) {
```

Comment:
Seems like with swift there is a preference for guard for semantic reasons but the end result is the same. [One opinion](https://www.natashatherobot.com/swift-guard-better-than-if/).  Since the work should never be done if the assignment results in nil, it would be "swiftier" to use guard and exit. 
Again, this was a nitpick so its up to you. :)